README.md

```diff
@@ -35,6 +35,22 @@ EventMachine.run do
 end
 ```
 
+To pass additional parameters while subscribing, add them to the identifier hash. These will be available server side in the channel's params hash.
```

Comment:
Hmm.

I think what would be super helpful for this gem is being able to have rspec test the connection with rails' action cable. (for another PR)

Because, I feel like this identity change is going to need a full integration test, ya know?